contracts/hardhat.config.ts

```diff
@@ -38,6 +38,42 @@ const getCommonNetworkConfig = (networkName: ESupportedChains, chainId: number,
   },
 });
 
+const getHardhatConfig = () => {
+  const defaultConfig = {
+    blockGasLimit: DEFAULT_BLOCK_GAS_LIMIT,
+    gasMultiplier: DEFAULT_GAS_MUL,
+    gasPrice: NETWORKS_DEFAULT_GAS[ESupportedChains.Hardhat],
+    chainId: EChainId.Hardhat,
+    accounts: {
+      mnemonic: TEST_MNEMONIC,
+      path: "m/44'/60'/0'/0",
+      initialIndex: 0,
+      count: 20,
+    },
+    gas: DEFAULT_BLOCK_GAS_LIMIT,
+    loggingEnabled: true,
```

Comment:
would probably keep this to false here too, and let users change it to true if neede